arrows/vxl/vidl_ffmpeg_video_input.cxx

```diff
@@ -120,11 +120,11 @@ class vidl_ffmpeg_video_input::priv
   bool d_have_metadata;
 
   double pts_of_meta_ts;            // probably seconds
```

Comment:
(Pedantic) These comments used to line up, though I don't see any particular value to that... maybe de-indent this comment also?